src/components/App/App.css

```diff
@@ -50,6 +50,16 @@ main {
   cursor: pointer;
 }
 
+@media screen and (max-height: 750px) {
+    .sidebar-nav {
+        margin-top: 15px;
+        position: static;
```

Comment:
I see that below, in another mediaquery, we are assigning `position: relative`. It will become messy soon.

Could we change the default position of `.sidebar-nav` to `position: relative` and then add a media query for
laptops to add the `position: fixed` so we **do mobile-first**?

Reply:
@tureey Sure... Will do it... 😇

Reply:
@tureey It's done now... Thanks. Would you like to approve?